src/tiled/objectselectionitem.cpp

```diff
@@ -147,6 +151,11 @@ class MapObjectOutline : public QGraphicsItem
 private:
     QRectF mBoundingRect;
     MapObject *mObject;
+
+    // Timer is used to call update function repeatedly to animate selection
+    QTimer *updateTimer = new QTimer();
```

Comment:
Since you're a `QObject`, please instead use `startTimer` and override `timerEvent`, so this can become `int mUpdateTimer;`.